From c871c86712a31c8dedfc62e2fb8093f1da3ec22b Mon Sep 17 00:00:00 2001
From: Devajith Valaparambil Sreeramaswamy
 <devajith.valaparambil.sreeramaswamy@cern.ch>
Date: Thu, 31 Oct 2024 09:32:18 +0100
Subject: [PATCH 20/51] Revert "Improve ROOT startup time by not merging
 identifier tables"

This reverts commit 5547e1e78739453fab0448cd4bdbfb0351afcba5.
---
 .../llvm-project/clang/lib/Serialization/ASTReaderInternals.h   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/interpreter/llvm-project/clang/lib/Serialization/ASTReaderInternals.h b/interpreter/llvm-project/clang/lib/Serialization/ASTReaderInternals.h
index abdea0d56b..25a46ddabc 100644
--- a/interpreter/llvm-project/clang/lib/Serialization/ASTReaderInternals.h
+++ b/interpreter/llvm-project/clang/lib/Serialization/ASTReaderInternals.h
@@ -46,7 +46,7 @@ class ASTDeclContextNameLookupTrait {
 
 public:
   // Maximum number of lookup tables we allow before condensing the tables.
-  static const int MaxTables = 256;
+  static const int MaxTables = 4;
 
   /// The lookup result is a list of global declaration IDs.
   using data_type = SmallVector<DeclID, 4>;
-- 
2.43.5

